hadoop-hdds/docs/dev-support/bin/generate-site.sh

```diff
@@ -31,11 +31,18 @@ if git -C $(pwd) status >& /dev/null; then
   ENABLE_GIT_INFO="--enableGitInfo"
 fi
 
-# Replace all markdown images with a responsive hugo shortcode.
-python3 $DIR/make_images_responsive.py $DOCDIR
+# Copy docs files to a temporary directory inside target
+# for pre-processing the markdown files.
+TMPDIR="$DOCDIR/target/tmp"
+mkdir -p "$TMPDIR"
+rsync -a --exclude="$DOCDIR/target" --exclude="$DOCDIR/public" "$DOCDIR/" "$TMPDIR"
```

Comment:
Thanks @JyotinderSingh for updating the patch.

Repeated `mvn package` increases the depth of the hierarchy (`target/tmp/target/tmp/target/tmp`, etc.).

```suggestion
mkdir -p "$TMPDIR"
rsync -a --exclude="target" --exclude="public" "$DOCDIR/" "$TMPDIR"
```

Reply:
Thank you for catching this, I have made the suggested changes.